app/javascript/sdk/IFrameHelper.js

```diff
@@ -43,6 +43,15 @@ const updateAuthCookie = cookieContent =>
     sameSite: 'Lax',
   });
 
+const updateCampaignReadStatus = () => {
+  const expireBy = new Date();
+  expireBy.setHours(expireBy.getHours() + 1);
+  Cookies.set('cw_snooze_campaigns_till', +expireBy, {
```

Comment:
I'm not sure if this is readable. What is the intent of using `+expireBy` instead of just `expireBy`